eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionReportWriter.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.pipeline.io;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.item.ExecutionContext;
+import org.springframework.batch.item.ItemStream;
+import org.springframework.batch.item.ItemStreamException;
+
+import uk.ac.ebi.eva.accession.core.ISubmittedVariant;
+import uk.ac.ebi.eva.accession.core.SubmittedVariant;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+// TODO make AccessionReportWriter implement ItemStreamWriter<AccessionWrapper<ISubmittedVariant, String, Long>>
+// when accession-commons 0.3 is used. Not worth to do it right now because
+// SubmittedVariantAccessioningService returns a Map.
+public class AccessionReportWriter implements ItemStream {
+
+    private static final String VCF_MISSING_VALUE = ".";
+
+    private static final String IS_HEADER_WRITTEN_KEY = "AccessionReportWriter_isHeaderWritten";
+
+    private static final String IS_HEADER_WRITTEN_VALUE = "true";   // use string because ExecutionContext doesn't support boolean
```

Comment:
I think that renaming this constant to TRUE would make more clear its content later in the code.

Reply:
that would explain the actual working, but would not give information on the purpose. this way both are expressed. I'm ok with improving the readability of the purpose, but I don't see how